CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+# 0.2.0 - 2017-12-21
+
+- use name to identify failpoint instead of absolute module path
+- support listing configured rules
```

Comment:
Logging was also changed: https://github.com/pingcap/fail-rs/commit/431f39269bd66365c2e19a358e01a0cdd7967a16

It might be worth noting. :)

Reply:
It has been changed before 0.1.0 is released.